zebrad/src/components/mempool.rs

```diff
@@ -35,12 +35,14 @@ pub use self::storage::tests::unmined_transactions_in_blocks;
 pub enum Request {
     TransactionIds,
     TransactionsById(HashSet<UnminedTxId>),
+    RejectedTransactionIds(HashSet<UnminedTxId>),
```

Comment:
The other 2 requests and response has tests:

https://github.com/ZcashFoundation/zebra/blob/main/zebrad/src/components/mempool/tests.rs#L19
https://github.com/ZcashFoundation/zebra/blob/main/zebrad/src/components/mempool/tests.rs#L30

I think it will be good to add a simple one for this new code as well.

Reply:
https://github.com/ZcashFoundation/zebra/pull/2718/commits/e68446fcebe17689da3ef818eb24d821c18ecf07